packages/cx-layout/src/components/Logo/LogoClean.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, HTMLProps } from 'react';
+
+import {
+  A,
+  DesignableComponentsProps,
+  Span,
+  Img,
+  designable,
+  ComponentOrTag,
+} from '@bodiless/fclasses';
+import {
+  asCxTokenSpec,
+} from '@bodiless/cx-elements';
+
+export type LogoComponents = {
+  Wrapper: ComponentOrTag<any>,
+  Image: ComponentOrTag<any>,
+  Link: ComponentOrTag<any>,
+};
+
+type LogoProps = DesignableComponentsProps<LogoComponents> & HTMLProps<HTMLElement>;
+
+const logoComponents: LogoComponents = {
+  Wrapper: Span,
+  Image: Img,
+  Link: A,
```

Comment:
Changing this to GatsbyLink would make Gatsby a dependency. Is this expected? I'm asking because I've heard a few conversations in the last few weeks about decoupling Bodiless functionality from Gatsby when possible.

Reply:
@e definitely don't want it in the clean component.  Maybe in a token... but maybe we should put thatin a separate @bodiless/vx-batsby package?